api/routes/user.py

```diff
@@ -0,0 +1,67 @@
+from fastapi import APIRouter, status, HTTPException, Depends
+from sqlalchemy.orm import Session
+from typing import List
+from schemas.user import UserCreate, UserInfo, UserBase
+from db.session import get_db
+from db.repository import user as repo
+
+router = APIRouter()
+
+
+@router.post("/", response_model=UserInfo, status_code=status.HTTP_201_CREATED)
+async def create_user(user: UserCreate, db: Session = Depends(get_db)):
```

Comment:
Usually, it’s better to use either `async def ` or `def` for all the endpoints

Reply:
Okay